%YAML:1.0
---

#相机内参矩阵K
camera_matrix: !!opencv-camera_matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 7.215377e+02, 0.0, 6.095593e+02,
          0.0, 7.215377e+02, 1.72854e+02, 
          0, 0, 1]


# 位移
distance_matrix: !!opencv-camera_matrix
   rows: 3
   cols: 1
   dt: d
   data: [4.485728e+01, 2.163791e-01, 2.745884e-03]



#lidar转换到相机的旋转矩阵
lidar_to_camera_R: !!opencv-camera_matrix
   rows: 3
   cols: 3
   dt: d
   data: [7.967514e-03, -9.999679e-01, -8.462264e-04,
         -2.771053e-03, 8.241710e-04, -9.999958e-01,
         9.999644e-01, 7.969825e-03, -2.764397e-03]

#lidar转换到相机的平移矩阵
lidar_to_camera_T: !!opencv-camera_matrix
   rows: 3
   cols: 1
   dt: d
   data: [-1.3777e-02, -5.5421e-02, -2.9185e-01]